contracts/BnomialToken1.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.2;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Burnable.sol";
+import "@openzeppelin/contracts/security/Pausable.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/draft-ERC20Permit.sol";
+import "@openzeppelin/contracts/token/ERC20/extensions/ERC20Votes.sol";
+
+contract BinomialToken1 is ERC20, ERC20Burnable, Pausable, Ownable, ERC20Permit, ERC20Votes {

```

Comment:
I'm not sure we need ERC20Permin and ERC20Votes for now. Let's try to keep things as simple as possible in the beginning. You can create Issues to evaluate ERC20Permin or ERC20Votes and potentially add them later.